runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -7323,56 +7324,6 @@ class INTERPRETER_CLASS
 		return rc;
 	}
 
-	/*
-	 * Determine if the two objects are substitutable
-	 *
-	 * @param[in] lhs the lhs object of acmp bytecodes
-	 * @param[in] rhs the rhs object of acmp bytecodes
-	 * return true if they are substitutable and false otherwise
-	 */
-	VMINLINE bool
-	acmp(j9object_t lhs, j9object_t rhs)
-	{
-#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
```

Comment:
Might be better to assign acmpResult to (lhs == rhs) initially, then test and modify it inside the ifdef so there's only a single return.

Reply:
Once again, I can't tell deletion from addition...